public/css/index.css

```
@@ -19,6 +19,6 @@
 
 @layer components, legacy, paper;
 
-@import url('../styles.css?v=95')      layer(components);
+@import url('../styles.css?v=99')      layer(components);
```

Stylelint (17.11.0)

[error] Expected "url('../styles.css?v=99')" to be "'../styles.css?v=99'" (import-notation)

(import-notation)